### PR TITLE
Add autoprefixer

### PR DESCRIPTION
**Pull request by brenoc**

### CODE REVIEW

**Comment by dfosco** [comment]
@brenoc Wait, was it working? I tested it and it was a no-go, but maybe I was mistaken...

**Comment by brenoc** [comment]
@dfosco I just created the PR, didn't tested yet

File: webpack.config.js [hunk 1]
```diff
@@ -33,8 +33,8 @@ var config = {
       }, {
         test: /\.scss$/,
         loaders: production ?
-          ['style', 'css', 'sass'] :
-          ['style', 'css?sourceMap', 'sass?sourceMap']
+          ['style', 'css', 'autoprefixer?browsers=last 2 version', 'sass'] :
+          ['style', 'css?sourceMap', 'autoprefixer?browsers=last 2 version', 'sass?sourceMap']
```
Comment (brenoc) [inline webpack.config.js]: You should add the autoprefixer in the development build as well

**Comment by brenoc** [comment]
@dfosco can you test it now?

**Comment by brenoc** [comment]
@dfosco I push forced this branch, so please fetch and reset you local branch

**Comment by dfosco** [comment]
Tested and working!

**Pull request closed without merging**